guestbook/php-redis/Dockerfile

```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM php:5-apache
+FROM BASEIMAGE
```

Comment:
The problem with that is additional maintenance.. BTW all the imaages including infra in `kubernetes/kubernetes` following this approach. @luxas any comments?